app/components/date-time-modal/date-time-modal-controller.tests.js

```diff
@@ -32,194 +35,189 @@ describe('Controller: DatetimeSelectionModalCtrl', function () {
             $scope: $scope,
             $modalInstance: $modalInstanceMock,
             CmsConfig: CmsConfig,
-            moment: moment
+            moment: moment,
           }
         );
       };
     });
+
+    today = moment();
+    tomorrow = moment().add(1, 'day');
+    dateFormat = 'YYYY-MM-DD';
+    timeFormat = 'HH:mm';
   });
 
   afterEach(function () {
     sandbox.restore();
   });
 
-  it('should set a timezone label on scope', function () {
+  it('has a TIMEZONE_LABEL', function () {
     CmsConfigProviderHook.setTimezoneName('America/Chicago');
-
     buildControllerInstance();
-
```

Comment:
I like to have spaces between setup, execution, and assertion for readability. Arguable, but I'd prefer to keep the spaced style to match newer tests.